index.html

```diff
@@ -1628,80 +1628,94 @@ <h4><dfn data-lt="updated" data-dfn-type="dfn" id="updated">updated</dfn></h4>
-[[DID-CORE]]
-              </td>
-            </tr>
-            <tr>
-              <td>
-<code style="white-space: nowrap;">version-id</code>
-              </td>
-              <td>
-Identifies a specific version of a DID document to be resolved (the
-version ID could be sequential, or a UUID, or method-specific). Note that
-this parameter might not be supported by all DID methods.
-              </td>
-              <td>
-[[DID-CORE]]
-              </td>
-            </tr>
+    <section>
+      <h4><dfn data-lt="service" data-dfn-type="dfn" id="service">service</dfn></h4>
+      <table class="simple" style="width: 100%;">
+        <thead>
+          <tr>
+            <th>Normative Definition</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <a href="http://w3c.github.io/did-core/#generic-did-url-parameters">DID Core</a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
 
-            <tr>
-              <td>
-<code style="white-space: nowrap;">version-time</code>
-              </td>
-              <td>
-Identifies a certain version timestamp of a DID document to be resolved.
-That is, the DID document that was valid for a DID at a certain
-time. Note that this parameter might not be supported by all DID methods.
-              </td>
-              <td>
-[[DID-CORE]]
-              </td>
-            </tr>
-          </tbody>
-        </table>
+      <pre class="example" title="Example of service parameter">
+did:example:123?service=agent
+      </pre>
+    </section>
+
+    <section>
+      <h4><dfn data-lt="version-id" data-dfn-type="dfn" id="version-id">version-id</dfn></h4>
+      <table class="simple" style="width: 100%;">
+        <thead>
+          <tr>
+            <th>Normative Definition</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <a href="http://w3c.github.io/did-core/#generic-did-url-parameters">DID Core</a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+      <pre class="example" title="Example of version-id parameter">
+did:example:123?version-id=4
+      </pre>
+    </section>
+
+    <section>
+      <h4><dfn data-lt="version-time" data-dfn-type="dfn" id="version-time">version-time</dfn></h4>
+      <table class="simple" style="width: 100%;">
+        <thead>
+          <tr>
+            <th>Normative Definition</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <a href="http://w3c.github.io/did-core/#generic-did-url-parameters">DID Core</a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+      <pre class="example" title="Example of version-time parameter">
+did:example:123?version-time=2016-10-17T02:41:00Z
+      </pre>
+    </section>
 
-    <p class="issue" data-number="32">
-      Example needed.
-    </p>
     <p class="issue" data-number="30">
       Example needed.
     </p>
```

Comment:
```suggestion
```

We can close this too since we're not differentiating 'extensions' in so many words any more.

Reply:
That sounds fine, but we could also add examples of extension DID parameters later (e.g. the `initial-values` one which is not in DID Core but still used by various DID implementations).

Reply:
good point